src/chocolatey.tests.integration/scenarios/ListScenarios.cs

```diff
@@ -496,8 +496,8 @@ public void should_contain_debugging_messages()
                 MockLogger.contains_message("Start of List", LogLevel.Debug).ShouldBeTrue();
                 MockLogger.contains_message("End of List", LogLevel.Debug).ShouldBeTrue();
             }
-        }        
-        
+        }

```

Comment:
Would prefer to see this whitespace change in a standalone commit.

Reply:
Sorted!